missing-hsts-header.cs

```diff
@@ -0,0 +1,83 @@
+public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+{
+    if (env.IsDevelopment())
+    {
+        // ruleid: missing-hsts-header
```

Comment:
Would it be possible to highlight just the function, or the first instance of the `app` instead of every time the app is used? As is, this will produce a lot of individual findings which can muddy up end users' UIs

Reply:
Thanks for the feedback @minusworld 

Can you please give me some guidance on how I can pick just the first instance of app?

Reply:
🤔 What about something like this? https://semgrep.dev/s/JRkd

This makes use of `focus-metavariable` to "zoom in" on the first use of the object in question, while still allowing you to highlight the object usage rather than the whole function.

Reply:
your suggestion @minusworld looks good. as discussed in our chat, would you mind making these changes for me please? thanks!